src/container/srv_container.c

```diff
@@ -1635,6 +1635,16 @@ cont_close_recs(crt_context_t ctx, struct cont_svc *svc,
 		rc = cont_iv_capability_invalidate(
 				svc->cs_pool->sp_iv_ns,
 				recs[i].tcr_hdl, CRT_IV_SYNC_EAGER);
+		if (rc == -DER_NONEXIST) {
```

Comment:
If the underlying CoRPC encountered two errors,

- one on rank X, who had the pool started, due to some network trouble, and
- one on rank Y, who didn't have the pool started,

then will the "group doesn't exist" error from rank Y hide the "network issue" error from rank X? If it will, when will rank X invalidate the capa in its memory?

Reply:
Overall I am not sure I understand these changes. 

If some node in the spanning tree happens to return DER_NONEXIST as it does not have the group then invalidate won't propagate past it to other nodes that will have the group. How is this condition being handled?

Isn't there some retry mechanism that should be kicking in to ensure invalidate request reaches all nodes eventually?

Reply:
If rank stopping will cause the IV not being broadcast to all nodes, then it will need retry indeed. Hmm, return -DER_OOG will cause iv to retry internally.

@liw if one rank is not part of the pool, then iv requests or corpc will not send rpc to this rank?  Not sure I follow the case here.  The intention here is to not fail for stopping node, but I am not sure which is the better choice, ignore or retry.